scenarios/remote_signals/src/main.rs

```diff
@@ -0,0 +1,238 @@
+use anyhow::Context;
+use holochain_types::prelude::*;
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use remote_signal_integrity::TimedMessage;
+use std::collections::HashSet;
+use std::sync::{Arc, Mutex};
+use std::time::Duration;
+use trycp_wind_tunnel_runner::embed_conductor_config;
+use trycp_wind_tunnel_runner::prelude::*;
+
+embed_conductor_config!();
+
+#[derive(Debug)]
+pub struct ScenarioValues {
+    signal_interval: Duration,
+    response_timeout: Duration,
+    remote_signal_peers: Vec<AgentPubKey>,
+    pending_set: Arc<Mutex<HashSet<TimedMessage>>>,
+}
+
+fn env_dur(n: &'static str, d: u64) -> Duration {
+    match std::env::var(n) {
+        Ok(n) => Duration::from_millis(n.parse::<u64>().unwrap()),
+        _ => Duration::from_millis(d),
+    }
```

Comment:
This seems like a [map_or](https://doc.rust-lang.org/std/option/enum.Option.html#method.map_or):

```suggestion
    std::env::var(n).map_or(Duration::from_millis(d), |ms| {
        Duration::from_millis(ms.parse::<u64>().unwrap())
    })
```

Reply:
It certainly could be! I personally find matches a lot more immediately understandable. And certainly when writing I don't have to remember, wait, did I want a `map_or`, `ok_or`, an `unwrap_or`... `and_then`?? I know some people really like the functional style, but it isn't me : )

Reply:
Fair enough :slightly_smiling_face: I completely understand and it's just personal preference.